src/bika/lims/browser/worksheet/views/referencesamples.py

```diff
@@ -248,11 +248,16 @@ def folderitem(self, obj, item, index):
         item["allow_edit"] = self.get_editable_columns()
 
         # Supported Services
-        supported_services_choices = self.make_supported_services_choices(obj)
-        item["choices"]["SupportedServices"] = supported_services_choices
+        supported_services = self.make_supported_services_choices(obj)
+        item["choices"]["SupportedServices"] = supported_services
+        item["SupportedServices"] = self.get_selected_items(supported_services)
 
         # Position
         item["Position"] = "new"
         item["choices"]["Position"] = self.make_position_choices()
 
         return item
+
+    def get_selected_items(self, supported_services_choices):
```

Comment:
Rename this function to `get_selected_values` to prevent missunderstandings with listing's `item` (note the functions `folderitem`, etc.) and change the name of the parameter`supported_services_choices` to `choices`

Reply:
ok, but line 253: `item["SupportedServices"] = self.get_selected_values(supported_services)` will be 80 characters size - PEP8:E501